### PR TITLE
[Bug 1247219] Fix Group settings UI elements

### PR DESCRIPTION
**Pull request by comzeradd**
* Fix Group title overflow text
* Limit select boxes width on mobile
* Proper arrangement of checkbox labels

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

**Comment by johngian** [comment]
Tested it locally

* Checkboxes are working fine
* Curators autocomplete field looks OK on complete
* Overflow text looks OK

**Comment by johngian** [comment]
Dropdown on curators auto-completion overflows field above.

**Comment by johngian** [comment]
Delete/resend invitation buttons are misaligned in mobile view (eg. 320x480)

**Comment by johngian** [comment]
Invites auto-completion field has the same issue that curators had.

**Comment by johngian** [comment]
I suggest we reduce the height between invites - checkbox and checkbox - textarea. @comzeradd what do you think?

selection_020

**Comment by comzeradd** [comment]
> Delete/resend invitation buttons are misaligned in mobile view (eg. 320x480)

Can you share a screenshot? If you mean that these are displayed in a new line that's expected. They stack under the name.

**Comment by johngian** [comment]
Is this expected? It's fine by  me if yes, but i thought this was misaligned:

selection_021

**Comment by comzeradd** [comment]
Yes that's expected. In most cases the name would be long enough anyway. It only looks kind of strange on very small names.

PR updated. I fixed the dropdown overflows. Unfortunately that needs an `!important` statement because the select2 js adds inline style. I also minimize the space between invitation email checkbox and textarea.

**Comment by johngian** [comment]
selection_022

Do you think that we need to add some padding here (check username)? The same applies for curators. (resolution 320x480)

**Comment by johngian** [comment]
Verified locally the rest of the changes and look OK.

**Comment by comzeradd** [comment]
Yes, you're right. I missed the help-block. I pushed a minor change to have the same width as the select input.

File: mozillians/static/mozillians/css/main.less [hunk 1]
```diff
@@ -2384,6 +2391,13 @@ form.add-group {
             padding-left: 0px;
         }
     }
+    #id_group_email_text {
+        margin-top: 0px;
+        padding-top: 0px;
+    }
+    .help-block {
+        max-width: 230px;
```
Comment (akatsoulas) [inline mozillians/static/mozillians/css/main.less]: This now forces the help text to break into multiple lines even when is not needed (eg full screen)

**Comment by comzeradd** [comment]
@akatsoulas PR updated

**Comment by akatsoulas** [comment]
Let's apply what you did in the custom email text in the other checkboxes too. r+wc

**Pull request merged**